app/controllers/api/v2/planning_elements_controller.rb

```diff
@@ -164,7 +165,7 @@ def find_all_projects_by_project_id
           # WTF. Why do we completely skip rewiring in this case and always provide parent_ids?
           # This is totally inconistent.
           identifiers = params[:ids].split(/,/).map(&:strip)
-          @planning_elements = WorkPackage.visible(User.current).find_all_by_id(identifiers)
+          @planning_elements = WorkPackage.visible(User.current).where(id: identifiers)
```

Comment:
This now returns an ActiveRecord::Relation. Don't know if that is a problem, though.